src/vs/editor/contrib/dropOrPasteInto/browser/defaultProviders.ts

```diff
@@ -200,6 +200,19 @@ class PasteHtmlProvider implements DocumentPasteEditProvider {
 	}
 }
 
+export class PasteImageProvider implements DocumentPasteEditProvider {
```

Comment:
This code should live under `workbench/chat` instead

Reply:
moved into it's own file `chatPasteProvider`, can maybe rename, since I don't think there are any other instances of providers being registered besides the `chatAccessibilityProvider`